apps/src/templates/CodeWorkspace.jsx

```diff
@@ -153,6 +134,11 @@ var CodeWorkspace = React.createClass({
     ];
   },
 
+  onToggleShowCode(usingBlocks) {
+    this.blockCounterEl.style.display =
+        (usingBlocks && studioApp.enableShowBlockCount) ? 'inline-block' : 'none';
```

Comment:
because there is still code in StudioApp which updates the contents of this counter via dom manipulation and I didn't want to mess that up by switching this section over.